pavelib/utils/test/suites/pytest_suite.py

```diff
@@ -163,15 +163,13 @@ def cmd(self):
             'pytest',
             '--ds={}'.format('{}.envs.{}'.format(self.root, self.settings)),
             "--junitxml={}".format(self.xunit_report),
+            '-vvv',
+            '-s',
+            '--create-db',
+            '--migrations',
```

Comment:
I think `--migrations` was the key to getting migrations to run, but not sure if it also needed `--create-db` or not.

Reply:
It was just `--migrations`.